### PR TITLE
Add ability to change Service type and exclude some k8s resources using vars  (Helm Chart)

### PR DESCRIPTION
**Pull request by azhurbilo**
- not create lighthouse resources if "lighthouse.enabled: true"
- ability not to use CA for postgresql
- ability to change Service type

#### :heavy_check_mark: Checklist

- [x] Added or updated documentation
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by Rugvip** [comment]
Thanks!

@martina-if, wanna have a look at this one? :grin: 🙏

File: contrib/chart/backstage/templates/lighthouse-deployment.yaml [hunk 1]
```diff
@@ -51,6 +51,7 @@ spec:
                   name: {{ include "lighthouse.postgresql.passwordSecret" . }}
                   key: postgresql-password
 
+      {{- if .Values.lighthouse.postgresCertMountEnabled }}
```
Comment (martina-if) [inline contrib/chart/backstage/templates/lighthouse-deployment.yaml:54]: nitpick: might be good for avoiding potential mistakes in future additions to this file to add two if-elses like in the backend part (one for lines 55-56 and another for  lines 59-61). Just a suggestion.

**Review by martina-if — APPROVED** [review]
LGTM!

File: contrib/chart/backstage/templates/lighthouse-deployment.yaml (hunk 1 above)
Comment (azhurbilo) [reply contrib/chart/backstage/templates/lighthouse-deployment.yaml]: added 2 "if" but before "volumeMounts" and "volumes" as there is only 1 element in the list, in other way we need add "[]" to "else" which also looks not good

Comment (martina-if) [reply contrib/chart/backstage/templates/lighthouse-deployment.yaml]: Right :+1:  looks good

**Review by martina-if — APPROVED** [review]

**Review by Rugvip — APPROVED** [review]
Alright, thanks! 😁

**Pull request merged**